kit/scrolls/src/Lists/UIMasonryList.tsx

```diff
@@ -335,6 +339,28 @@ function useVirtualization<Item>(
         [initialWindowHeight, onLayoutProp, setWidth],
     );
 
+    const maybeCallOnEndReached = React.useCallback(
+        (contentLength: number, visibleLength: number) => {
+            const distanceFromEnd = contentLength - visibleLength;
+            const threshold =
+                onEndReachedThreshold != null ? onEndReachedThreshold * contentLength : 2;
```

Comment:
Can you add a comment that here we replicate the VirtualizedList behaviour, and the default value of `2` is taken from the SectionList documentation:
https://reactnative.dev/docs/sectionlist#onendreachedthreshold

Btw, in my understanding a threshold should be calculated a bit differently according to the documentation, i.e the `onEndReachedThreshold` is just a multiplier to the visible length of the list (which is equal to windowHeight):
```suggestion
            const threshold = windowHeight * (onEndReachedThreshold ?? 2);
```